developer/architecture/layout.md

```diff
@@ -1,12 +1,93 @@
+
+_Workflow processes_ and _workflow steps_ are serving different needs, which is why they have different properties (a lot of them are not shown in the above example for clarity reasons). However, from a UI perspective the difference is that the _workflow process_ acts as a parent container element for the _workflow step_ children. This also implies that a _View_ can accommodate more than one _Layout_. Obviously, that's happening in the example above, because there are multiple, individual _workflow steps_ declared.
+
+
+### Workflow process properties
+`workflow`
+* Name for the workflow. Correlates with a _View_ specified in registry. See also the [Routes section](https://docs.reactioncommerce.com/reaction-docs/master/plugin-routes-6)
+
+`layout`
+* This property's name is a bit misleading. Conceptually it designates a shop wide identifier for a collection of _Layouts_ (subsequently referred to only as **_Global Layout_**), because it covers potentially many routes (as opposed to _Layout_, which only refers to a specific _View_)
+
+`collection`
+* Refers to a mongo collection which holds the _workflow state_, if any.
+
+`structure`
+* Contains information about which components/widgets should be rendered when this _workflow process_ is applied by the Reaction router.
+* `structure.template`
+  * The name of the [Blaze](http://docs.meteor.com/#/full/blaze_render) template that renders the main content for this _workflow process_.
+* `structure.notFound`
+  * Name of Blaze template that gets rendered if `structure.template` does not exist (Should not be necessary).
+* `structure.headerLayout`
+  * The name of the React component that renders the header for this _workflow process_.
+* `structure.footerLayout`
+  * The name of the React component that renders the footer for this _workflow process_.
+
+`enabled`
+  * Wether this _workflow process_ should be enabled or not.
+
+### Workflow step properties
+`workflow`
+* Name for the workflow. Correlates with a _View_ specified in registry. See also the [Routes section](https://docs.reactioncommerce.com/reaction-docs/master/plugin-routes-6)
+
+`container`
+* The DOM element where the rendered template should go into.
+
+`template`
+  * The name of the [Blaze](http://docs.meteor.com/#/full/blaze_render) template that renders the content for this _workflow step_.
+
+`enabled`
+  * Wether this _workflow step_ should be enabled or not
+
+### Notices
+* At any given time, only one _Global Layout_ can be active. See also `layout` property above.
+
+* At any given time - within the active _Global Layout_ - only one of the related workflow processes can be active.
+
+### Implementation details:
+
+* Layouts are meant to be created using the Package Registry. During initialization, the _workflow processes_ are cloned to the Shops collection which is where the application refers to them at runtime. Contrary, the _workflow steps_ are not cloned to the Shops collection.
```

Comment:
rather than `Contrary` we should probably say something like `On the other hand`